src/twisted/mail/relaymanager.py

```diff
@@ -841,11 +845,11 @@ def _ebExchange(self, failure, factory, domain):
         @type domain: L{bytes}
         @param domain: A domain.
         """
-        log.err('Error setting up managed relay factory for ' + domain)
+        log.err(b'Error setting up managed relay factory for ' + domain)
```

Comment:
Same comment as above about leaving the text as text and converting the `domain` to textual form for logging